containers/server-systemd-services/uyuni-server.service

```diff
@@ -0,0 +1,37 @@
+[Unit]
+Description=Uyuni server image container service
+Wants=network.target
+After=network-online.target
+After=postgresql.service
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/etc/sysconfig/uyuni-server-systemd-services
+Restart=on-failure
+ExecStart=/usr/bin/podman run --tmpfs /run \
+    -p 443:443 -p 80:80 -p 4505:4505 -p 4506:4506 \
+    -v /sys/fs/cgroup:/sys/fs/cgroup:rw \
+    -v pgsql:/var/lib/pgsql \
+    -v spacewalk:/var/spacewalk \
+    -v cache:/var/cache \
+    -v share-susemanager:/usr/share/susemanager \
+    -v share-salt-formulas:/usr/share/salt-formulas \
```

Comment:
the share volumes are not needed. See the volumes I mount in the rke2 pod definition: https://github.com/uyuni-project/uyuni/blob/server-container/containers/doc/server-rke2/server.yaml#L141